osbs/build/build_request.py

```diff
@@ -465,16 +465,16 @@ def set_kerberos_auth(self, plugins):
             self.dj.dock_json_set_arg(phase, plugin, 'koji_principal', krb_principal)
             self.dj.dock_json_set_arg(phase, plugin, 'koji_keytab', krb_keytab)
 
-    def set_reactor_config_map(self):
-        if not self.spec.reactor_config_map.value:
+    def set_reactor_config_map(self, reactor_config_map):
```

Comment:
Let's rebase with master as this method has changed.

Reply:
Fixed.